.vscode/launch.json

```diff
@@ -4,18 +4,24 @@
   // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```

Comment:
We should add this file to the .gitignore so we don't accidentally commit it. It's not needed in our repo, but we don't necessarily want to delete it off our computers. To add this to the .gitignore, find the file called `.gitignore` in the root of the repository. At the end of that file, add the following:

```
.vscode/**
```

Then, delete this file. Git should detect it's gone and remove it from the repo.